CMakeLists.txt

```diff
@@ -42,6 +42,7 @@ set (HMI "web" CACHE STRING "HMI type")
 option(EXTENDED_MEDIA_MODE "Turn on and off extended Madia Manager features relates to PulseAudio A2DP and GStreamer" OFF)
 option(BUILD_SHARED_LIBS "Build all libraries as shared (if ON) or static (if OFF)" OFF)
 option(BUILD_BT_SUPPORT "Bluetooth support" ON)
+option(BUILD_BLE_SUPPORT "Bluetooth Low Energy support" ON)
```

Comment:
```suggestion
option(BUILD_BLE_SUPPORT "Bluetooth Low Energy support" OFF)
```

Reply:
fixed in 8c504abbb37320d77c18a8662a47b5a61942b7cf